tools/oracle2bq-schema-converter/example_terraform_dir/terraform copy.tfvars

```diff
@@ -0,0 +1,94 @@
+project_id                  = "project_id"
+  default_table_expiration_ms = null
+  dataset_labels = {
+    env      = "dev"
+    workload = "hr"
+    owner    = "sametkaradag"
+  }
+
+tables = [
+  {
+    table_id = "COUNTRIES",
+    schema   = "schemas/COUNTRIES.json",
+    time_partitioning = null
+    expiration_time = null,
+    clustering      = null,
+    labels = {
```

Comment:
why have table labels identical to dataset labels?
IMHO table labels should provide strictly more granular infomation than dataset labels.